### ISSUE TITLE
Crashing when accessing storage 

### ISSUE THREAD

**Issue by guppy42** [guppy42/mcpelauncher-linux#1]
version: 1.1.0.4 beta

**Comment by guppy42** [comment]
```internal storage path = data/private/
internal storage path = data/private/
__bionic_open_tzdata_path: ANDROID_DATA not set!
__bionic_open_tzdata_path: ANDROID_ROOT not set!
__bionic_open_tzdata_path: ANDROID_DATA not set!
__bionic_open_tzdata_path: ANDROID_ROOT not set!
__bionic_open_tzdata_path: ANDROID_DATA not set!
__bionic_open_tzdata_path: ANDROID_ROOT not set!
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
Signal 6 received
Backtrace elements: 10
#0 ./mcpelauncher(_Z12handleSignaliPv+0x77) [0x805f897]
#1 [0xf7793bc0]
#2 [0xf7793be9]
#3 /lib/i386-linux-gnu/libc.so.6(gsignal+0x39) [0xf72b5ea9]
#4 /lib/i386-linux-gnu/libc.so.6(abort+0x157) [0xf72b7407]
#5 HYBRIS __gnu_cxx::__verbose_terminate_handler()+469 in libminecraftpe.so+0x253f6f5 [0xf64ca6f5]
#6 HYBRIS __cxxabiv1::__terminate(void (*)())+25 in libminecraftpe.so+0x24dff49 [0xf646af49]
#7 HYBRIS __cxxabiv1::__unexpected(void (*)())+0 in libminecraftpe.so+0x24dffe0 [0xf646afe0]
#8 HYBRIS (null)+89 in libminecraftpe.so+0x24e0199 [0xf646b199]
#9 HYBRIS void std::vector<LocalWorldInfo, std::allocator<LocalWorldInfo> >::_M_emplace_back_aux<LevelSummary&>(LevelSummary&)+416 in libminecraftpe.so+0xd97320 [0xf4d22320]
Dumping stack...
```

**Comment by guppy42** [comment]
Also true for 1.1.0.9 the error seems to occur when ever there are world that would populate the world list.

Maybe it's related to the new thumbnail generating for the world list?

**Comment by guppy42** [comment]
Fixed my @MCMrARM in the 1.1 branch
